src/app/static/src/js/components/runWorkflow/runWorkflow.vue

```diff
@@ -10,6 +10,7 @@
                          :submit-label="toggleFinalStepNextTo"
                          @cancel="handleCancel"
                          @complete="handleComplete"
+                         @changeTab="handleChangeTab"
```

Comment:
I'd prefer this event (and associated methods and events, here and in other components) to be more descriptive of the intention, rather than the switching of tabs, which is actually just one part of what needs to happen once fully implemented (showing the progress page with the correct workflow selected). So maybe 'viewWorkflowProgress', and include the workflow key as a parameter in the event. 

I've just been implementing something similar, but going in the opposite direction, in https://github.com/vimc/orderly-web/pull/366 Switching tab is part of what happens ([see here](https://github.com/vimc/orderly-web/pull/366/files#diff-eaf22518fa1c5b6343983553d6fc93735c16494c2f871c138d6ca8a70aea6f0cR76-R78)), but it also sets a property on child component - I would expect this feature to be similar in approach.